pom.xml

```diff
@@ -173,6 +213,18 @@
 					<autoReleaseAfterClose>false</autoReleaseAfterClose>
 				</configuration>
 			</plugin>
+
+			<!-- executes unit tests -->
+			<plugin>
```

Comment:
If this isn't getting any special configuration, why explicitly list it?

Reply:
It was required to have the tests run when executing `mvn test` or any lifecycle phase that calls it.  Without the surefire plugin stated explicitly, the tests didn't run when I executed `mvn test`.

Reply:
Oh, really? I thought it used to run by default unless the `-DskipTests` option was given. Was this required for unit tests or integration tests?

Reply:
Or... maybe the default behaviour has changed...

Reply:
I thought so, too.  The tests were run by default for some of the other projects but not others like this one.  I'm not sure why, but explicit surefire configuration was able to get them working.  I may come back to take a closer look into what's going on, though.

For this project, all the tests are unit tests.  Running both `mvn test` and `mvn clean package` without the surefire plugin included produced output that no tests were run but maven still ran to completion successfully.

Reply:
Really? Weeeeeeird. Well, if adding the plugin works, go for it! :)

Reply:
> Or... maybe the default behaviour has changed...

I did see something online about changes to the maven surefire plugin, but I think something else may be going on that is specific to the projects that didn't run their tests by default.

Reply:
> Really? Weeeeeeird. Well, if adding the plugin works, go for it! :)

Yeah, I would like to figure out why.  It makes me uneasy that it didn't work or not work consistently without the maven surefire plugin specified explicitly.